Makefile

```diff
@@ -19,7 +19,7 @@ $(eval PI = $(shell cat /proc/cpuinfo 2>&1 | grep "Raspberry Pi"))
 all: clippy build test
 
 .PHONY: cross-ci
-cross-ci: cross-clippy cross-build-aarch64 cross-test-aarch64
+cross-ci: cross-clippy cross-build-armv7 cross-test-armv7
```

Comment:
Let’s change it so it does cross for both arch?
Either that or two make targets (one for each arch), and have two GH Action jobs, one for arm7 and one for aarch64 - but will be slightly quicker if same job does both (only one install of cross etc)